v2/bench_test.go

```diff
@@ -47,3 +55,93 @@ func BenchmarkReadBlocks(b *testing.B) {
 		}
 	})
 }
+
+// BenchmarkExtractV1File extracts inner CARv1 payload from a sample CARv2 file using ExtractV1File.
+func BenchmarkExtractV1File(b *testing.B) {
+	path := filepath.Join(b.TempDir(), "bench-large-v2.car")
+	generateRandomCarV2File(b, path, 10*1024*1024) // 10 MiB
```

Comment:
neat trick if you weren't aware: `10 << 20 // 10 MiB`.

`<< 10` for KiB, `<< 20` for MiB, `<< 30` for GiB, and so on. Easier to type and read, if you ask me, as the "metric scale" is a number that increases 1-2-3-etc.

also pretty intuitive once you remember that `1 << 10 == 1024`